### PR TITLE
Update KillAura.kt

### PR DESCRIPTION
**Pull request by ll11l1lIllIl1lll**
IDK

### CODE REVIEW

**Comment by opZywl** [comment]
no like this update u removed direction mode lol

**Comment by ll11l1lIllIl1lll** [comment]
> no like this update u removed direction mode lol

rename to FOV

**Comment by opZywl** [comment]
fix NoSwing Killaura and InfiniteAura 
@ll11l1lIllIl1lll

**Comment by ll11l1lIllIl1lll** [comment]
@opZywl I didn't find the problem in KillAura swing

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/KillAura.kt [hunk 1]
```diff
@@ -277,7 +281,7 @@ class KillAura : Module() {
         if (rotationStrafeValue.equals("Off"))
             update()
 
-        if (target != null && currentTarget != null) {
+        if (target != null && currentTarget != null && ( (attackTimingValue.get()=="Pre" && event.eventState == EventState.PRE) || (attackTimingValue.get()=="Post" && event.eventState == EventState.POST) || attackTimingValue.get()=="Both" || attackTimingValue.get()=="All")) {
```
[the remark this answers is not in the document]
Comment (opZywl) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/combat/KillAura.kt:284]: > @opZywl I didn't find the problem in KillAura swing

the problem is just visual for you, don't show people what's right is people can never see

**Pull request merged**